README.md

```diff
@@ -48,6 +48,14 @@ Contributions where also made by
 
 ## Release notes
 
+### 3.1.0 (2022-10-21)
+
+* Improve compatibility with PHP 8.1;
+  in particular, the new `__serialize`/`__unserialize` methods are implemented now
+  (in addition to the still supported `Serializable` interface).
+  Care has been taken to keep the output of `getHash()` stable;
+  other classes that include data values in their own hashes may want to use the new `getSerializationForHash()` method where it exists.
```

Comment:
Should maybe say "other classes that include data values **php serialization** in their own hashes"... In all other cases, we probably want people to actually include our hashes directly.

Reply:
Done (with some other rephrasing in the sentence).